gcc/rust/ast/rust-expr.h

```diff
@@ -4640,6 +4658,10 @@ struct InlineAsmRegOrRegClass
     std::string Symbol;
   };
 
+  Type type;
+  struct Reg reg;
+  struct RegClass regClass;
```

Comment:
nit: we usually go for snake case in class members
```suggestion
  struct RegClass reg_class;
```